synorchestrator/trs/client.py

```diff
@@ -115,3 +127,13 @@ def get_workflow_files(self, id, version_id, type, fix_url=True):
             id, version_id, type
         )
         return _get_endpoint(self, endpoint)
+
+    def post_verification(self, id, version_id, type, relative_path, requests):
```

Comment:
@jaeddy let us know if it would be helpful to actually hook this up 
@garyluu could just ignore failures or print a warning until we roll out the new dockstore version with the live endpoint